src/styles/index.scss

```diff
@@ -1,3 +1,120 @@
+@import "./utils/variables\ ";
+
+html {
+  font-family: Roboto, sans-serif;
+}
+
 body {
   margin: 0;
 }
+
+.card {
+  width: 198px;
+  height: 406px;
+  background-color: $color-white;
+  border: 1px solid #f3f3f3;
+  border-radius: 5px;
+  margin: 20px 48px;
+
+  &__image {
+    width: 160px;
+    height: 134px;
+    padding: 32px 19px;
+
+  }
+
+  &__name {
+    font-size: 12px;
+    font-weight: 500;
+    line-height: 18px;
+    padding: 4px 16px;
+    color: $color-dark;
+  }
+
+  &__code {
+    font-size: 10px;
+    font-weight: 400;
+    line-height: 14px;
+    padding-left: 16px;
+    color: #616070;
+  }
+}
+
+.stars {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+
+  &__stars {
+    display: flex;
+    padding: 16px;
+  }
+
+  &__star {
+    background-image: url('../images/star.svg');
+    width: 16px;
+    height: 16px;
+    background-repeat: no-repeat;
+    background-position: center;
+  }
+
+  &__star:not(:last-of-type) {
+    margin-right: 4px;
+  }
+
+  &__star:nth-child(-n + 4) {
```

Comment:
The stars quantity should depend on the modifier

Reply:
<img width="651" alt="image" src="https://github.com/mate-academy/layout_product-cards/assets/70220667/122bccde-9547-47fd-8479-adc08c13c747">